timelock-server/build.gradle

```diff
@@ -39,11 +45,44 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-core'
     testCompile project(":atlasdb-tests-shared")
 
-    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
 }
 
 distribution {
     serviceName "timelock-server"
     mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
     args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-Xms512m",
+            "-Xmx512m",
+            "-javaagent:service/lib/jetty-alpn-agent-${libVersions.jetty_alpn_agent}.jar"
+}
+
+dependencyRecommendations {
+    strategy ConflictResolved
+}
+
+configurations.all {
```

Comment:
nit: In general seems like we could move this logic into dependencyRecommendations but if it's working, I don't see much point in changing it now.  The current format is reasonably readable.